packages/internal-build-utils/scripts/create-azure-sdk-for-net-pr.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+/**
+ * Standalone script to create PR in azure-sdk-for-net repository
+ * This script reads arguments from command line and creates a PR to update the http-client-csharp dependency
+ */
+
+import { fileURLToPath } from "url";
```

Comment:
Let's do something similar to this - https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/pipelines/templates/archetype-typespec-emitter.yml#L355-L431